edx_sga/sga.py

```diff
@@ -491,6 +495,16 @@ def download_zip(self, student_files):
         # save zip file and return its URL as JSON response
         return Response(json={'zip_url': default_storage.url(default_storage.save(zip_filename, sio))})
 
+    def get_submission_download_status(self, student_id):
+        student_item_dict = self.student_submission_id(student_id)
+        staff_user = User.objects.get(id=self.xmodule_runtime.user_id)
```

Comment:
You can dry this into a property.
Then you can avoid doing this here and just send something like `self.user` when you call `get_download_status` or `set_as_downloaded`.